### ISSUE TITLE
Collision/navigation issues on short stair model still exists on alt texture versions

### ISSUE THREAD

**Issue by Cerebulon** [TD-Addon/TamrielDataMain#379]
Same issue as #356 applies to alt texture versions of the same model. Fix was applied to tr/x/ex_de_docks_steps_02.nif, but should be applied to:

hr/x/hr_ex_docks_steps_02.nif
hf/x/hf_ex_docks_steps_02.nif
pc/i/pc_in_platfc_steps03.nif
pc/x/pc_ex_docks_steps_02.nif
pc/x/pc_ex_platf_steps03.nif
sky_ex_docks_steps_03.nif

**Comment by CdoubleOK** [comment]
Rotated collision back 1 degree to allow pathing

[Step fixes.zip](https://github.com/user-attachments/files/25731210/Step.fixes.zip)
